src/components/about/About.scss

```diff
@@ -0,0 +1,35 @@
+.about {
+    background-color: rgba(8, 11, 209, 0.712);
+    min-height: 100vh;
+  }
+  
+  .about__wrap {
+  
+    background-color: #fff;
+  }
+  
+  .about__content {
+      display: flex;
+      width: 100%;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      padding: 5% 0 5% 0;
+  }
+
+.about__content-wrap {
+  display: flex;
+  width: 60%;
+  flex-direction: column;
+  justify-content: center;
+}
+
+
+.aboout__title {
+  margin-bottom: 5%;
```

Comment:
margins/paddings with `%` look a little bit weird (sometimes it's ok, but very rarely), could it be done in another way?

Reply:
yup, replaced with rem.